pkg/server/commands/check_command.go

```diff
@@ -74,7 +74,7 @@ func WithCacheController(ctrl cachecontroller.CacheController) CheckQueryOption
 func NewCheckCommand(datastore storage.RelationshipTupleReader, checkResolver graph.CheckResolver, typesys *typesystem.TypeSystem, opts ...CheckQueryOption) *CheckQuery {
 	cmd := &CheckQuery{
 		logger:             logger.NewNoopLogger(),
-		datastore:          datastore,
+		datastore:          storagewrappers.NewMetricsOpenFGAStorage(datastore),
```

Comment:
Store this in a var above, so the interface doesn't need to change, and subsequently have you have access to the additional methods you require.

Reply:
If i have the command object keep a reference to both `datastore` and `instrumentedDatastore`, then i wouldn't reference the former anywhere 🤔